css/styles.css

```diff
@@ -328,3 +328,115 @@ ul li {
   align-items: center;
   text-align: center;
 }
+
+content {
+  width: 320px;
+  height: 363px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: rgba(23, 43, 77, 1);
+  margin-top: 50px;
+}
+
+.contact-me {
+  background: url(../img/Frame.png);
+  background-position: right 62%;
+  background-repeat: no-repeat;
+  margin-top: 30px;
+  padding-top: 40px;
+}
+
+.forms {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 0;
+  padding: 15px;
+  font-family: 'Inter', sans-serif;
+}
+
+input:focus {
+  outline: none;
+}
+
+.forms1 {
+  display: block;
+  width: 100%;
+  height: 48px;
+  border-radius: 4px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(208, 217, 212, 1);
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.forms2 {
+  height: 114px;
+  border: 1px solid rgba(96, 192, 149, 1);
+  margin-bottom: 30px;
+}
+
+.forms2:focus {
+  outline: none;
+}
```

Comment:
Kindly change all these class names to a more descriptive class. `forms1` and `forms2` is not.